packages/web-app-external/src/components/ResourceNameModal.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <form autocomplete="off" @submit.prevent="onConfirm">
+    <oc-text-input
+      id="save-as-input-file-name"
+      v-model="newFileName"
+      class="oc-mb-s"
+      :label="$gettext('File name')"
+      required-mark
+      :error-message="errorMessage"
+      :fix-message-line="true"
```

Comment:
In other modals, we "mark" the name, this is missing here

Reply:
Ah, file name selection in the input. FTR, we solved that together now ❤️